tests/unit_tests/test_game.py

```diff
@@ -390,33 +422,388 @@ def custom_game(event_loop, database, game_service, game_stats_service):
 
 async def test_compute_rating_balanced_teamgame(game: Game, player_factory):
     game.state = GameState.LOBBY
-    players = [
-        (player_factory(**info), result, team) for info, result, team in [
-            (dict(login='Paula_Bean', player_id=1, global_rating=Rating(1500, 250.7)), 0, 1),
-            (dict(login='Some_Guy', player_id=2, global_rating=Rating(1700, 120.1)), 0, 1),
-            (dict(login='Some_Other_Guy', player_id=3, global_rating=Rating(1200, 72.02)), 0, 2),
-            (dict(login='That_Person', player_id=4, global_rating=Rating(1200, 72.02)), 0, 2),
-        ]
-    ]
+    players = [(player_factory(**info), result, team)
+               for info, result, team in [
+                   (
+                       dict(
```

Comment:
Can we condense this maybe by using positional arguments and `enumerate` for the playerid/login. The login name doesn't matter here so how about we just use the id as the login. I'm thinking something like:

```python
players = [
    (i, player_factory(f"Player {i}", rating), result, team) 
    for i, rating, result, team in enumerate([
        (Rating(1500, 200), 0, 1)
    )]
]
```
(not sure the order of the parameters)

Especially because this is just data, I feel like it doesn't need to take up this much space.

Reply:
Ah, I overlooked your suggestion and instead I gave constant names to the few distinct players that we were using. But I like yours better, since the constant names a) make it hard to tell the rating of players from the test body and b) force the players in different tests to vary together, although they should be independent. I'll adopt your suggestion.